### ISSUE TITLE
holland returning true when xtrabackup fails

### ISSUE THREAD

**Issue by soulen3** [holland-backup/holland#267]
Looks like it's possible for holland to return true even when the backup fails. I found this while testing PR #265, but it doesn't appear to be related.

```
~
    ! InnoDB: the problem may be that during an earlier attempt you managed
    ! InnoDB: to create the InnoDB data files, but log file creation failed.
    ! InnoDB: If that is the case, please refer to
    ! InnoDB: http://dev.mysql.com/doc/refman/5.6/en/error-creating-innodb.html
Backup Failed to run
Backup failed after 0.12 seconds
Purged xtrabackup/20190205_094118
1 backups purged
Released lock /etc/holland/backupsets/xtrabackup.conf
--- Ending backup run ---
# echo $?
0
```

**Comment by soulen3** [comment]
This bug was caused by 7a4e6cae67e5159d096e36f9e548192774ede61c.  So the BackupError exception wasn't being captured by the `sys.exc_info()` call.

**Comment by soulen3** [comment]
Fixed in 1.1.12
